internal/cel/library/net_test.go

```diff
@@ -0,0 +1,95 @@
+package library
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+
+	"github.com/google/cel-go/cel"
+	"github.com/kubewarden/policy-sdk-go/pkg/capabilities"
+	"github.com/kubewarden/policy-sdk-go/pkg/capabilities/net"
+	"github.com/stretchr/testify/require"
+)
+
+func TestNet(t *testing.T) {
+	tests := []struct {
+		name           string
+		expression     string
+		response       []string
+		expectedResult interface{}
+	}{
+		{
+			"kw.net.lookupHost",
+			"kw.net.lookupHost('kubewarden')",
+			[]string{"192.168.0.1", "10.0.0.1"},
+			[]string{"192.168.0.1", "10.0.0.1"},
+		},
+		{
+			"kw.net.lookupHost test return type",
+			"kw.net.lookupHost('kubewarden')[0]",
+			[]string{"192.168.0.1", "10.0.0.1"},
+			"192.168.0.1",
```

Comment:
I think we can just keep the second test case

Reply:
well, I will leave both around, just for the doc side of things with the testcase name.